S17-supply/zip.t

```diff
@@ -44,7 +44,24 @@ for ThreadPoolScheduler.new, CurrentThreadScheduler -> $*SCHEDULER {
 
     throws-like( { Supply.zip(42) },
       X::Supply::Combinator, combinator => 'zip' );
-
 }
 
+subtest {
+    my $*SCHEDULER = ThreadPoolScheduler.new;
+    react {
+        whenever Promise.in(1) {
+            flunk "zip timed out";
+            done;
+        }
+        whenever Supply.zip: Supply.from-list(^2), Supply.interval: .01 -> $l {
+            if $l eqv ((0, 0) | (1, 1)) {
+                pass "zip is ok"
```

Comment:
I think this could use a better description: "zip terminates when the first of its input supplies terminates".

Reply:
do you think that for the pass message? or on subtest message?

Reply:
Oh, I just noticed there's a better one on the subtest; I missed it entirely. So, it's fine. :-)